frontend/editor.css

```diff
@@ -1,4 +1,6 @@
 @import url("https://cdn.jsdelivr.net/npm/dialog-polyfill@0.5.6/dist/dialog-polyfill.css");
+@import url("https://cdn.jsdelivr.net/npm/bcp47-picker@1.0.16/dist/bcp47-picker.css");
+@import url("https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css");
```

Comment:
this one is pretty big 🤔 (165kb)

Reply:
yeah i also don't want it. i wish bcp47 had a bundle that included all the styles that it needs, scoped to the element but i think we need to do that by hand

Reply:
this is one place where the PR got stuck